### ISSUE TITLE
E488: Trailing characters

### ISSUE THREAD

**Issue by XSven** [WolfgangMehner/vim-plugins#52]
I have installed this plugin (5.5alpha) with vim-plug plugin manager. My vim version is 8.1.2130. The operating system is AIX. The perl version is 5.14.2. When I open a Perl module I am getting the Errors

```
Error detected while processing function Perl_InitializePerlInterface:
line    7:
E488: Trailing characters: ^I^I^I# ---------------------------------------------------------------
line    8:
E488: Trailing characters: ^I^I^I# find out the version of the Perl interface
line    9:
E488: Trailing characters: ^I^I^I# ---------------------------------------------------------------
line   13:
E488: Trailing characters: ^I^I^I# ---------------------------------------------------------------
line   14:
E488: Trailing characters: ^I^I^I# Perl_RegexExplain (function)
line   15:
E488: Trailing characters: ^I^I^I# try to load the regex analyzer module; report failure
line   16:
E488: Trailing characters: ^I^I^I# ---------------------------------------------------------------
line   24:
E171: Missing :endif
```
Any help to fix this is very much appreciated.

**Comment by WolfgangMehner** [comment]
Sorry for the late response. Does the problem persist? Is it present with other versions of Vim?

From the error messages it seems like Vim tries to interpret the Perl code as Vim script. Is Perl correctly compiled into your editor?

**Comment by XSven** [comment]
Yes the problem does still exist. Unfortunately I have no access to other versions of Vim. I think Perl is not correctly complied into the Vim editor. The perl feature is off. I guess that this is the reason. Sorry. One question before I am asking our admins to recompile Vim with perl feature enabled: The perl feature seems to be an "m"type  (manually enabled or depends on other features). I could not figure out what that means!

**Comment by WolfgangMehner** [comment]
About the error:

I'm slightly confused about the behavior of our plug-in here. We check for the Perl capabilities before we execute the code which causes the above problems. With your Vim, what is the result of the following commands:
```
:echo has('perl')
:perl print("eee")
```

About manual features:

Vim comes with various predefined sets of features you can choose from when compiling it yourself: tiny, small, normal, big, ... Some features are not enabled by any of these configurations. You must configure them manually before compiling, e.g.:
```
./configure --enable-pythoninterp=yes
```

**Comment by XSven** [comment]
`:echo has('perl')` prints 0 and `:perl print("eee")` simply does nothing. To enable the perl feature we have to run configure with `--enable-perl=yes` right?

**Comment by WolfgangMehner** [comment]
Should be `--enable-perlinterp=yes`.

**Comment by XSven** [comment]
I think we have to close or flag this issue. Our admins refuse to build from source and the packaged versions of vim they are allowed to install have the perl feature disabled :-( I will try to compile from source by my own but I don't know when. Sorry and thanks so far.

**Comment by WolfgangMehner** [comment]
No worries.
